<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Request</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="requestMethod">
         <item>
          <property name="text">
           <string>GET</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>POST</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PUT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DELETE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string notr="true">Body</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Content-Type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="requestBody"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="requestContentType">
             <item>
              <property name="text">
               <string>application/json</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>application/x-www-form-urlencoded</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>multipart/form-data</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QOpenGLWidget" name="openGLWidget"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string notr="true">Headers</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="requestHeaders"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string notr="true">Cookies</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="requestCookies"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="requestUrl"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Response</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string notr="true">Body</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTabWidget" name="tabWidget_3">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_7">
              <attribute name="title">
               <string notr="true">Pretty</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPlainTextEdit" name="responseBodyPretty"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_8">
              <attribute name="title">
               <string notr="true">Raw</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPlainTextEdit" name="responseBodyRaw"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_9">
              <attribute name="title">
               <string notr="true">Web view</string>
              </attribute>
              <layout class="QVBoxLayout" name="responseBodyWebLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string notr="true">Headers</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPlainTextEdit" name="responseHeaders"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string notr="true">Cookies</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPlainTextEdit" name="responseCookies"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
